matter_server/common/models/node.py

```diff
@@ -0,0 +1,133 @@
+"""Matter node."""
+from __future__ import annotations
+from dataclasses import dataclass, field
+from datetime import datetime
+
+from typing import TYPE_CHECKING, Dict
+import logging
+
+from .device_types import RootNode, Aggregator, ALL_TYPES, BridgedDevice
+from chip.clusters import Objects as Clusters, Attribute
+
+from ..helpers.util import dataclass_to_dict
+from .device_type_instance import MatterDeviceTypeInstance
+from .node_device import (
+    AbstractMatterNodeDevice,
+    MatterBridgedNodeDevice,
+    MatterNodeDevice,
+)
+
+from chip.clusters import (
+    Attribute,
+    Cluster,
+    ClusterObject,
+    ClusterAttributeDescriptor,
+    ClusterEvent,
+    ClusterObjects,
+)
+
+LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class MatterNode:
```

Comment:
What's the difference between this `MatterNode` and the one in `common/models/device.py`?

Reply:
Can we do the cleanup in a follow-up PR and instead focus on merging + getting it working in HA?